### ISSUE TITLE
Packages built by Docker are broken

### ISSUE THREAD

**Issue by rustprooflabs** [rustprooflabs/pgdd#19]
# Details

I thought the packaging in Docker was working again but it isn't.  I think it's related to a change in pgx paths, the Docker image expects a Pg version in the path e.g. `pg13`.  Now it's getting `13.4`.
 

## Posted to Discord

Working against develop branch -- It seems the paths in the built extension now include the Pg minor version instead of only major version.   Example in the same branch of docs shows `pg12` in the path. https://github.com/zombodb/pgx/tree/develop/cargo-pgx#building-an-installation-package

```bash
installing extension
     Copying control file to `target/release/spi-pg12/usr/share/postgresql/12/extension/spi.control`
```

Now running my extension I get `13.4` in the path instead o f `pg13`.  I don't think I did anything silly to change this but... :person_shrugging:

```bash
installing extension
     Copying control file to `/home/notyouruser/.pgx/13.4/pgx-install/share/postgresql/extension/pgdd.control`
```

Do I need to adjust my packaging to become minor version aware?

----

**Comment by rustprooflabs** [comment]
The Docker process here did have at least one minor issue and definitely can still use some improvement.  Though seems the core of the problem documented here is in pgx from some recent changes, they're on it! :+1:

https://github.com/zombodb/pgx/issues/204

**Comment by rustprooflabs** [comment]
The issue should most mostly resolved after 4bb1d3824cf6fae195f6a1a96fcfaa02986e79fd.  That seems to have solved the packaging related issues though cannot confirm yet due to another likely bug.

Talked with Hoverbear via Discord who managed to determine my computers are duplicating files through some arcane wizardry :mage_woman:.

> I think what's happening is somehow in https://github.com/zombodb/pgx/blob/d9c1db535bb5bd27f6cb70217866a040f3f98efb/cargo-pgx/src/commands/schema.rs#L136-L173 some things are being found twice
> So I will also probably make a PR to dedupe that list for you to try
> I could see this happening because your particular system may create multiple archive objects (line 146) which could somehow have the same named symbols (l148) which could be pushed onto this list
> You can see here: https://github.com/zombodb/pgx/blob/d9c1db535bb5bd27f6cb70217866a040f3f98efb/cargo-pgx/src/commands/schema.rs#L248-L255 That list gets sent via an env variable

**Comment by rustprooflabs** [comment]
Resolved via #20.  Remove bionic, replaced with hirsute.
